modules/restful_token_auth/restful_token_auth.module

```diff
@@ -49,8 +49,16 @@ function restful_token_auth_restful_parse_request_alter(&$request) {
   $plugin = restful_get_authentication_plugin('token');
   $param_name = $plugin['options']['param_name'];
 
+  $value = \RestfulManager::getRequestHttpHeader($param_name);
+  if (!$value && !empty($_GET[$param_name])) {
```

Comment:
No, because the $request might be the values from the ``$_POST``, and we want to explicetly check also the ``$_GET`` for the presence of access_token